lib/nostrum/api.ex

```diff
@@ -1255,12 +1255,9 @@ defmodule Nostrum.Api do
 
   ## Examples
 
-  ```Elixir
-  Nostrum.Api.modify_guild(81384788765712384, name: "Elixir Guild")
-  {:ok, %Nostrum.Struct.Channel{id: 81384788765712384, name: "Elixir Guild"}}
-
-  Nostrum.Api.modify_channel(81384788765712384)
-  {:ok, %Nostrum.Struct.Channel{id: 81384788765712384}}
+  ```elixir
+  Nostrum.Api.modify_guild(451824027976073216, name: "Nose Drum")
+  {:ok, %Nostrum.Struct.Guild{id: 451824027976073216, name: "Nose Drum"}}
```

Comment:
```suggestion
  {:ok, %Nostrum.Struct.Guild{id: 451824027976073216, name: "Nose Drum", ...}}
```
How do you feel about something like this to indicate there are more fields that we're just not showing?